models/service/hubspot__daily_ticket_history.sql

```diff
@@ -0,0 +1,144 @@
+{{
+    config(
+        enabled=var('hubspot_service_enabled', False),
+        materialized='incremental',
+        partition_by = {'field': 'date_day', 'data_type': 'date'}
+            if target.type not in ['spark', 'databricks'] else ['date_day'],
+        unique_key='id',
```

Comment:
Worlds smallest request. Can we renamed the unique key to be more specific to the models in question. So I would imagine `ticket_history_id` or something like that.

Reply:
updated to ticket_day_id